.github/workflows/test-action.yml

```diff
@@ -0,0 +1,129 @@
+name: Test GitHub Action
+
+on:
+  pull_request:
+    paths:
+      - 'action.yml'
+      - '.github/workflows/test-action.yml'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  test-action:
+    name: Test Action on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Setup wasixcc with all components
+        uses: ./
+        with:
+          version: 'latest'
+          install-sysroot: 'true'
+          install-llvm: 'true'
+          install-binaryen: 'true'
+      
+      - name: Verify wasixcc installation
+        shell: bash
+        run: |
+          echo "Testing wasixcc..."
+          wasixcc --version
+          
+          echo "Testing wasix++..."
+          wasix++ --version || echo "wasix++ not found"
+          
+          echo "Testing wasm-opt..."
+          wasm-opt --version || echo "wasm-opt not found"
```

Comment:
I don't think wasm-opt should be added to the path. remove this line in the test